Fix/v4 compatibility

gltarsa opened the pull request.

pdf commented:
Can you confirm whether split navigation and dropdowns still function correctly with v4.x?  Please drop the version bump commit - I'll need to bump the major version since we'll be changing the simple-navigation dependency, and there is a rake task that will handle this for release.

gltarsa commented:
If by "split navigation" you mean "sub-navigation," then yes, both features still function correctly with v4.x because I am using them both in our code.  I have reverted the version file to the current master version, so only two files are now changed.

pdf commented:
If you take a look at the README, there's a split navigation feature that creates a separate dropdown toggle, so that the primary button can function with a link.  I just have no way to test this easily right now.

gltarsa commented:
Working on this now.  I have run into an unrelated problem with my app during the test.  I'll let you know shortly when the test is complete.

gltarsa commented:
The test will not be happening tonight.

pdf commented:
No problem, take your time.

pdf commented on simple-navigation-bootstrap.gemspec:
```diff
@@ -20,4 +20,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "simple-navigation", ">= 4.0.0"
   s.add_runtime_dependency "railties", ">= 3.1"
+  # thor dependencies assume the spurious warning messages bug will be fixed 19.5
+  s.add_runtime_dependency "thor", "0.19.1", "!=0.19.2", "!=0.19.3", "!=0.19.4"
```
Should probably only be a development dependency.

pdf commented:
Do you suspect this is close to ready for review?
